src/base/matrix.rs

```diff
@@ -1930,6 +1981,24 @@ impl_fmt!(fmt::UpperHex, "{:X}", "{:1$X}");
 impl_fmt!(fmt::Binary, "{:b}", "{:.1$b}");
 impl_fmt!(fmt::Pointer, "{:p}", "{:.1$p}");
 
+/// Displays a vector using commas as the delimiter
+pub fn format_column_vec_as_row<T: Scalar + fmt::Display, D: crate::DimName>(
+    vector: &OVector<T, D>,
+    f: &mut fmt::Formatter<'_>,
+) -> fmt::Result
+where
+    DefaultAllocator: Allocator<T, D>,
+{
+    write!(f, "[")?;
+    let mut it = vector.iter();
+    std::fmt::Display::fmt(it.next().unwrap(), f)?;
```

Comment:
Would be nice not to panic on 0-element vectors, unless we're really utterly certain that will never occur (but the signature doesn't suggest as much).

Reply:
0-length vectors can easily occur in practice, so this is important!

Would be nice to have some unit tests that include 0-element vectors maybe..

Reply:
I've added a test for 0-element matrices, though I'm not sure how to add a test for this function directly. Since calling that function requires a `Formatter` struct, and I don't think I can construct a 0-length translation/point to test with.